src/custom/pages/Profile/styled.tsx

```diff
@@ -129,12 +130,14 @@ export const FlexCol = styled.div`
   justify-content: center;
   strong {
     font-size: 21px;
+    line-height: 40px;
```

Comment:
I agree with [Michel's comment](https://github.com/gnosis/cowswap/pull/1604#issuecomment-943104600), we should control the spacing by using the margin (instead of increasing the line-height).
For example: margin-top: 6px